index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Tribute Page</title>
+  <link href="https://fonts.googleapis.com/css2?family=Comfortaa:wght@300&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./style.css">
+</head>
+
+<body>
+  <div id="main">
+    <h1 id="title">Nelson Mandella</h1>
+    <div id="img-div">
+      <img id="image" src="https://www.aljazeera.com/mritems/Images/2020/2/10/11c14071534440a1bc2146f0cd1a3d69_8.jpg"
+        alt="Mandella's image" />
+      <div id="img-caption">
+        Nelson Mandela in Soweto two days after his release from prison. He
+        addressed more than 100,000 people inside a football stadium. 'During
+        the past 27 years I have looked forward to this day when I would come
+        back to the area I regard as home, to meet my brothers and sisters and
+        grandchildren.'
+      </div>
+    </div>
```

Comment:
@zieeco  It would be better if you use your comment, as suggestions instead of orders and you should Kindly explain why you required this change, it might be obvious for you but not for someone else.
for example : I see that you are using a `<div>` tag as the main section I suggest changing it to a [`<main>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main) tag that will  make your website more accessible by the screen readers and  improve your SEO benchmark